pkgs/top-level/all-packages.nix

```diff
@@ -19601,7 +19601,8 @@ in
 
     rtl8821cu = callPackage ../os-specific/linux/rtl8821cu { };
 
-    rtlwifi_new = callPackage ../os-specific/linux/rtlwifi_new { };
+    rtw88 = if lib.versionAtLeast kernel.version "4.14" then callPackage ../os-specific/linux/rtw88 { } else null;
+    rtlwifi_new = rtw88;
```

Comment:
This should be moved to aliases.nix with a throw.

Reply:
Moving this to aliases.nix doesn't work as it is a part of the `linuxPackagesFor` function, so I changed the already existing line to a throw.